editors/vscode/client/extension.ts

```diff
@@ -70,7 +69,8 @@ export async function activate(context: ExtensionContext) {
   const traceOutputChannel = window.createOutputChannel(traceOutputChannelName);
 
   const ext = process.platform === "win32" ? ".exe" : "";
-  const command = join(context.extensionPath, `./target/release/oxc_vscode${ext}`);
+  let serverPath = process.env.SERVER_PATH_DEV ?? `oxc_vscode${ext}`;
```

Comment:
We should have a language server path config to let user customize, this could be improve in next pr.